lib/options.nix

```
@@ -187,6 +189,188 @@ rec {
   */
   mkPackageOptionMD = mkPackageOption;
 
+  /* Creates a suggestion based resource namespace allocator.
+
+    It allows someone to define a value space, like a port interval, so other
+    modules can "ask" for a value and the module will check for missing values,
+    conflicts and invalid candidates and suggest values for these cases that
+    must be explicitly set.
+
+    See RFC 0159 for more specification details.
+  */
+  mkAllocModule = {
```

For the good sake of naming, 
```suggestion
  mkAllocatorModule = {
```

---

Ok